.github/workflows/main.yml

```diff
@@ -0,0 +1,59 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - '*'
+  pull_request:
+    branches:
+      - master
+      - '*'
```

Comment:
Same change as the other one where you don't call out `master` and `*`?

Reply:
Ack